types/ember/index.d.ts

```diff
@@ -2637,3 +2638,358 @@ declare namespace Ember {
 }
 
 export default Ember;
+}
+
+declare module '@ember/application' {
+    import Ember from 'ember';
+    export default class Application extends Ember.Application { }
```

Comment:
Interesting; this also [fails the lint](https://travis-ci.org/typed-ember/ember-typings/builds/264796866#L558):

> strict-export-declare-modifiers  'export' keyword is redundant here.